src/const/NonPrintableKeysDictionary.js

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
🐥 New dictionary of non-printable keys, that takes its values from the same list that the React normalization algorithm uses.